client/src/components/Home.svelte

```diff
@@ -1,17 +1,11 @@
 <script>
   import { onMount } from "svelte";
-
+  import { Link } from "svelte-routing";
 
 
   let loggedIn = false;
   let perks = [];
 
-  async function fetchPerks() {
-      const response = await fetch("/api/perks");
-      perks = await response.json();
-  }
-
-  fetchPerks();
```

Comment:
be sure to add a replacement for fetching perks from the perkManager (I understand this was removed due to issues with the perkManager file)